package.json

```diff
@@ -71,60 +71,60 @@
     "node": "^10.12.0 || >=12.0.0"
   },
   "devDependencies": {
-    "@babel/code-frame": "^7.12.13",
-    "@babel/parser": "^7.14.6",
-    "@babel/types": "^7.14.4",
+    "@babel/code-frame": "^7.14.5",
+    "@babel/parser": "^7.15.5",
```

Comment:
It seems that updating `babel` dependencies caused our AST alignment tests to fail
https://github.com/typescript-eslint/typescript-eslint/pull/3859/checks?check_run_id=3553713788#step:9:19

@bradzacher I'm happy to fix these, if you could point me into the right direction

Reply:
they have been fixed in #3833 - rebase against master.

Reply:
Rebased 🙂